resources/carrier/zh/86.txt

```diff
@@ -0,0 +1,52 @@
+# Copyright (C) 2016 The Libphonenumber Authors
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Translations of en/86.txt from each carrier's wikipedia page and in
+# some cases the carrier's own home page.
+
+86130|中国联通
+86131|中国联通
+86132|中国联通
+86133|中国电信
+86134|中国移动
+86135|中国移动
+86136|中国移动
+86137|中国移动
+86138|中国移动
+86139|中国移动
+86145|中国联通
+86147|中国移动
+86150|中国移动
+86151|中国移动
+86152|中国移动
+86153|中国电信
+86155|中国联通
+86156|中国联通
+86157|中国移动
+86158|中国移动
+86159|中国移动
```

Comment:
There are some recent changes to the en file, https://github.com/googlei18n/libphonenumber/pull/1260 - can you add them too? Thanks!